src/cli/flags/sharepoint.go

```diff
@@ -8,26 +8,30 @@ const (
 	DataLibraries = "libraries"
 	DataPages     = "pages"
 	DataLists     = "lists"
+
+	InvalidListTemplate = "webTemplateExtensionsList"
```

Comment:
This is the wrong ownership for a variable like this.  It should be owned by the api package.

Reply:
yes @ryanfkeepers 
In this [PR](https://github.com/alcionai/corso/pull/4870), I have put it in api package and intended to remove it from here post merges.